.phpstan/bootstrap.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class ProjectConfiguration extends sfProjectConfiguration
+{
+}
+
+class BaseForm extends sfForm
+{
+}
+
+include 'PEAR/REST/10.php';
```

Comment:
:see_no_evil:  why do we need PEAR?! :see_no_evil: 

Should we completely drop it? @thirsch IIRC you mentioned something like this?

Reply:
plugin uses this. as you can see in my plugin loading improvement issue #342 i would drop it too. if only because the default symfony channels don't work. refer to propel discussion